examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -15,7 +14,8 @@
       "vmSize": "Standard_D2_v2",
       "OSDiskSizeGB": 200,
       "vnetSubnetId": "/subscriptions/SUB_ID/resourceGroups/RG_NAME/providers/Microsoft.Network/virtualNetworks/VNET_NAME/subnets/SUBNET_NAME",
-      "firstConsecutiveStaticIP": "10.239.255.239" 
+      "firstConsecutiveStaticIP": "10.239.255.239",
+      "vnetCidr": "10.230.0.0/16"
```

Comment:
This is the kubernetes api model that we feed as input to the ginkgo E2E runner. Added `vnetCidr` as a way of converting this api model over to an Azure CNI w/ custom VNET cluster.